tests/test_solvermode/test_species_model.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core.examples import download_file
+
+
+@pytest.fixture(scope="module")
+def solver_session():
+    cas_path = download_file(
+        "mixing_elbow.cas.h5",
+        "pyfluent/mixing_elbow",
+    )
+    solver_session = pyfluent.launch_fluent(mode="solver")
+    solver_session.file.read_case(file_name=cas_path)
+    yield solver_session
+    solver_session.exit()
```

Comment:
We have usually been specifying fixtures in separate files as they are re-used in multiple places, but as this one is specific to this test module, having it here makes sense to me too. 

Also, good that you are re-using one session for multiple tests, rather than launching a separate session for each test

Reply:
Yeah, I created a specific fixture for this because the original mixing elbow fixture is function-scoped, but I needed a module-scoped fixture to run multiple tests on the same solver object.